hnn_core/tests/test_gui.py

```diff
@@ -19,6 +22,19 @@
 from ipywidgets import Tab, Text, link
 
 matplotlib.use('agg')
+hnn_core_root = Path(__file__).parents[1]
+assets_path = Path(hnn_core_root, 'tests', 'assets')
+
+
+@pytest.fixture
+def setup_gui():
+    gui = HNNGUI()
+    gui.compose()
+    gui.params['N_pyr_x'] = 3
+    gui.params['N_pyr_y'] = 3
+    gui.widget_dt.value = 0.5  # speed up tests
```

Comment:
```suggestion
    gui.widget_dt.value = 0.85  # speed up tests
```
If we are just testing the GUI functionality and not data consistency, we can use a larger dt.